### PR TITLE
Last updated date

### PR DESCRIPTION
**Pull request by AdamZarger**
Displayed the `last_updated` date as a subtitle of the Search title within the app. This date comes from Elasticsearch's `_meta` property and is calculated at runtime. This completes DPT-609

## Additions

- `last_updated` date

## Review

- @JeffreyMFarley 
- @sephcoster 

## Todo

- Fix broken unit tests

## Checklist

* [ ] Changes are limited to a single goal (no scope creep)
* [ ] Code can be automatically merged (no conflicts)
* [ ] Code follows the standards laid out in the [front end playbook](https://github.com/cfpb/front-end)
* [ ] Passes all existing automated tests
* [ ] New functions include new tests
* [ ] New functions are documented (with a description, list of inputs, and expected output)
* [ ] Placeholder code is flagged
* [ ] Visually tested in supported browsers and devices
* [ ] Project documentation has been updated (including the "Unreleased" section of the CHANGELOG)

### CODE REVIEW

File: src/SearchPanel.jsx [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
+      utcDate.getMonth(),
+      utcDate.getDate() + 1
+    )
 
+    var tryagain = shortIsoFormat( utcDate );
 
-export default SearchPanel
+    return (
+      <div className="search-panel">
+        <h2>Search complaint data <span className="date-subscript">(last updated: <FormattedDate value={ tryagain } />)</span></h2>
```
Comment (JeffreyMFarley) [inline src/SearchPanel.jsx]: You should probably have a code path for when the last updated date is not available (like when the search fails, for example)

This will also help reduce the testing profile since the 'null' path would be executed by other components since they typically won't know to set up this variable

File: src/__tests__/__snapshots__/App.spec.jsx.snap [hunk 2]
```diff
@@ -97,7 +97,15 @@ exports[`initial state renders without crashing 1`] = `
     <div
       className="search-panel">
       <h2>
-        Search complaint data
+        Search complaint data 
+        <span
+          className="date-subscript">
+          (last updated: 
+          <span>
+            12/31/1969
```
Comment (JeffreyMFarley) [inline src/__tests__/__snapshots__/App.spec.jsx.snap]: This date __always__ means an uninitialized date variable

File: src/SearchPanel.jsx [hunk 3]
```diff
@@ -8,25 +8,24 @@ import { shortIsoFormat } from './utils'
...
-    var tryagain = shortIsoFormat( utcDate );
-
-    return (
-      <div className="search-panel">
-        <h2>Search complaint data <span className="date-subscript">(last updated: <FormattedDate value={ tryagain } />)</span></h2>
-        <SearchBar />
-        <PillPanel />
-      </div>
-    )
+    if (this.props.lastUpdated) {
```
Comment (JeffreyMFarley) [inline src/SearchPanel.jsx]: You don't have to rewrite the whole HTML block, the code could be simplified to:

```Typescript
let e =  null
if ( this.props.lastUpdated ) {
  e = <span className="date-subscript">(last updated: <FormattedDate value={ this.props.lastUpdated } />)</span>
}

...

<h2>Search complaint data { e }</h2>
```

File: src/SearchPanel.jsx [hunk 4]
```diff
@@ -8,24 +8,24 @@ import { shortIsoFormat } from './utils'
 
 export class SearchPanel extends React.Component {
   render() {
-    if (this.props.lastUpdated) {
+    if ( this.props.lastUpdated ) {
```
Comment (JeffreyMFarley) [inline src/SearchPanel.jsx]: You don't have to rewrite the whole HTML block, the code could be simplified to:

```Typescript
let e =  null
if ( this.props.lastUpdated ) {
  e = <span className="date-subscript">(last updated: <FormattedDate value={ this.props.lastUpdated } />)</span>
}

...

<h2>Search complaint data { e }</h2>
```

**Comment by AdamZarger** [comment]
Closing this PR in favor of this one instead: https://github.com/cfpb/ccdb5-ui/pull/90
